dist/README.UPDATERS

```diff
@@ -1,12 +1,31 @@
 For Updaters from OBS 2.9 to OBS 2.10
 =====================================
 
-Note: Update from OBS 2.5 should also work, but is untested.
-      A direct update from OBS 2.4 or older will not work.
-      It's recommended to wait for all pending delayed jobs to be processed before
-      creating a DB dump or running the migrations.
-      To see the number of pending delayed jobs run
-      `(cd /srv/www/obs/api; bundle exec rails r -e production "puts Delayed::Job.count")`
+Note: * Update from OBS 2.5 should also work, but is untested.
+        A direct update from OBS 2.4 or older will not work.
+        It's recommended to wait for all pending delayed jobs to be processed before
+        creating a DB dump or running the migrations.
+        To see the number of pending delayed jobs run
+        `(cd /srv/www/obs/api; bundle exec rails r -e production "puts Delayed::Job.count")`
+
+Note: * The format of the OBS options.yml is now distinguishing between Rails environments.
```

Comment:
I think we need to make this easier somehow (conversion rake tasks maybe?) and/or provide a fallback as this is a major change.

Reply:
I'll provide a script or rake task:+1: